### ISSUE TITLE
Add Balloon image to QGC

### ISSUE THREAD

**Issue by kaklik** [ThunderFly-aerospace/TF-B1#2]
The balloon is already available at [PX4 airframes reference](https://docs.px4.io/master/en/airframes/airframe_reference.html#balloon). But it would be useful to [add it in QGC also](https://docs.px4.io/master/en/airframes/airframe_reference.html#balloon).  The required steps are: 

- [ ] Compile the QGC with an embedded Balloon image to be able to  take a screenshot to pull-request
- [x] Edit image to enhance rope visibility in QGC airframes list.

**Comment by roman-dvorak** [comment]
Nemůže být provázek modrý, jako gondola?

**Comment by kaklik** [comment]
Gondola ale modrá není..   Teď jsem vymyslel tohle řešení: 

![image](https://user-images.githubusercontent.com/1194177/106738773-a5b0ae80-6618-11eb-8f8c-38352642b816.png)

Přidal jsem ho do [forku QGC](https://github.com/ThunderFly-aerospace/qgroundcontrol/tree/add_TF-B1_airframe). Zřejmě je to ale potřeba vyzkoušet zkompilovat a já nemám nikde zprovozněné build prostředí pro QGC.

**Comment by roman-dvorak** [comment]
Tohle by bylo nejlepší udělat PR přimo do QGC a pak to ověřit z jejich daily buildu.

**Comment by kaklik** [comment]
> Tohle by bylo nejlepší udělat PR přimo do QGC a pak to ověřit z jejich daily buildu.

jo, jenže oni v tom pull-requestu minule chtěli screenshot toho, že to tam je..
